### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by thomassth** [Eonasdan/tempus-dominus#2782]
### Prerequisites

- [X] I have [searched](https://github.com/Eonasdan/tempus-dominus/issues?q=label%3A%22Type%3A+feature+request%22+) for duplicate or closed feature requests.
- [X] I have read the [contributing guidelines](https://github.com/Eonasdan/tempus-dominus/blob/master/.github/CONTRIBUTING.md).

### Proposal

One of the biggest regression from v5 to v6 is the lack of keyboard navigation within the datepicker. 

For example, I can't pick dates in calendar with arrow keys.

### Motivation and context

See W3C ARIA Authoring Practices Guide for more details and an example
https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/

**Comment by Eonasdan** [comment]
this is being tracked with https://github.com/Eonasdan/tempus-dominus/issues/2390
